src/library/cmd_reader.h

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
Did you base/reference you implementation on the CRT implementation?

Reply:
Yeah, Ben sent me the original code.

Reply:
Yeah, Ben sent me the original code.